src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/AccountSupervisor.kt

```diff
@@ -912,7 +912,7 @@ internal open class AccountSupervisor(
             state?.transferStatuses,
             state?.restriction,
             state?.launchIncentive,
-            Compliance(state?.compliance?.geo, compliance.status, compliance.updatedAt),
+            compliance,
```

Comment:
Wait no, it's like this because geo is actually not set in AccountSupervisor, but instead set here:
https://github.com/dydxprotocol/v4-abacus/blob/main/src/commonMain/kotlin/exchange.dydx.abacus/state/v2/manager/StateManagerAdaptorV2.kt#L674

So if we just set compliance like this, geo would get over written to null.

but agree the code is not great, broken up like this. There is probably a better way to do it 🤔